deploy/kubernetes/v1.12+/attacher.yaml

```diff
@@ -78,7 +81,7 @@ spec:
             capabilities:
               add: ["SYS_ADMIN"]
             allowPrivilegeEscalation: true
-          image: quay.io/k8scsi/csi-attacher:v0.4.1
+          image: quay.io/k8scsi/csi-attacher:v1.0-canary
```

Comment:
Do you know what's the difference between v1.0.0 and v1.0-canary?

Reply:
Yep, v1.0.0 has a bug where it reports the wrong version: https://github.com/kubernetes/kubernetes/issues/71378

There's not a v1.0.1 yet, but v1.0-canary is supposed to get all minor updates.